server/filters/get-days.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import moment from 'moment';
 
-export default function formatDate(date: Date): string {
+export default function getDays(date: Date): string {
```

Comment:
the point of format date was to have 1 date formatter to rule them all to keep style consistent.
Could we try stick to that? Or are there different requirements?